jwcrypto/jwe.py

```diff
@@ -8,6 +8,17 @@
 from jwcrypto.common import json_decode, json_encode
 from jwcrypto.jwa import JWA
 
+__all__ = [
+    'JWEHeaderRegistry',
```

Comment:
this is not really public

Reply:
Why? if I need to register a new header, for example, `exp` in order to limit the lifetime, and specify it for mandatory processing in the `crit` list.

Reply:
See the PR that landed recently, there is now a mechanism to provide your own custom headers.
In fact you need to rebase the PR as that one breaks yours.

Reply:
You are talking about https://github.com/latchset/jwcrypto/pull/148
Yes, now it is enough `JWSEHeaderParameter` from the module `common`